lib/node_modules/@stdlib/stats/incr/mprod/lib/main.js

```diff
@@ -52,7 +52,7 @@ function product( workspace, acc, x ) {
 
 	// Ensure fraction remains normalized to avoid overflow/underflow...
 	if ( acc.frac > -0.5 && acc.frac < 0.5 ) {
-		frexp( workspace, acc.frac );
+		frexp( acc.frac, workspace );
```

Comment:
```suggestion
		frexp( acc.frac, workspace, 1, 0 );
```

Missing stride and offset.